neuraxle/steps/data.py

```diff
@@ -94,9 +92,10 @@ class EpochRepeater(MetaStepMixin, BaseStep):
         :class:`BaseStep`
     """
 
-    def __init__(self, wrapped, epochs, fit_only=True):
+    def __init__(self, wrapped, epochs, fit_only=True, repeat_in_test_mode=False):
         BaseStep.__init__(self)
         MetaStepMixin.__init__(self, wrapped)
+        self.repeat_in_test_mode = repeat_in_test_mode
```

Comment:
Ah, I see. Then at least, even if repeat_in_test_mode is a default argument to False, it'd be good to give it in the docstring of my other comment above to remove any ambiguity for the reader (e.g.: I was confused when commenting above).